ui/package.json

```diff
@@ -132,23 +136,19 @@
     "ember-cli-flash": "4.0.0",
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-mirage": "2.4.0",
-    "ember-cli-page-object": "1.17.10",
+    "ember-cli-mirage": "^3.0.2",
+    "ember-cli-page-object": "^2.1.3",
     "ember-cli-sass": "11.0.1",
     "ember-cli-sri": "meirish/ember-cli-sri#rooturl",
     "ember-cli-string-helpers": "6.1.0",
     "ember-cli-terser": "^4.0.2",
-    "ember-cli-typescript": "^5.2.1",
     "ember-composable-helpers": "5.0.0",
-    "ember-concurrency": "2.3.4",
-    "ember-copy": "2.0.1",
-    "ember-d3": "^0.5.1",
+    "ember-concurrency": "^3.1.1",
     "ember-data": "~4.11.3",
     "ember-engines": "0.8.23",
     "ember-fetch": "^8.1.2",
     "ember-inflector": "4.0.2",
     "ember-load-initializers": "^2.1.2",
-    "ember-maybe-in-element": "^2.0.3",
```

Comment:
This probably took the longest to figure out. I kept getting the error `unable to use this non-serializable babel config in this node process` but the stack trace wasn't very helpful.

This issue: https://github.com/embroider-build/embroider/issues/1525 is the same as I was seeing, and adding the `throwUnlessParallelizable: true,` as described in that thread didn't print the name of the addon, it just listed "unknown". So after some trial and error and thinking about what addons likely have babel transforms, I tried removing this, and it fixed the build 🎉.

`ember-maybe-in-element` won't be necessary anymore as recent ember has the `~in-element` helper included. So this is safe to remove.